src/components/bookMeeting/ContractorsList.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import ContractorCard from './ContractorCard'; // Importuj komponent ContractorCard
+import './ContractorsList.css'; // Dodaj stylizację dla listy kontraktorów
+
+const ContractorsList = () => {
+	const [contractors, setContractors] = useState([]);
+
+	useEffect(() => {
+		// Pobierz dane kontraktorów z endpointu GET
+		fetch('http://localhost:5142/api/contractors')
```

Comment:
There is a proper place for all the url in src/data catalogue, links.ts. Consider importing it. In case path changes we do not have to check all the files, only links.ts.

Reply:
@marcioszka Please check if now is okey after my last commit